bangc-ops/mlu_op.h

```diff
@@ -7432,6 +7432,203 @@ mluOpDeformRoiPoolBackward(const mluOpHandle_t handle,
+mluOpGetIndiceConvolutionBackwardDataWorkspaceSize(mluOpHandle_t handle,
+                                                   const mluOpTensorDescriptor_t output_grad_desc,
+                                                   const mluOpTensorDescriptor_t filters_desc,
+                                                   const mluOpTensorDescriptor_t indice_pairs_desc,
+                                                   const mluOpTensorDescriptor_t input_grad_desc,
+                                                   const int64_t indice_num[],
+                                                   const int64_t inverse,
+                                                   size_t *workspace_size);
+
+// Group:IndiceConvolutionBackwardData
+/*!
+ * @brief Performs the backpropagation of a indice convolution operation to
+ * compute the gradient of input \b input_grad based on the gradient of response
+ * \b output_grad, the filter tensor \b filter, the indice tensor \b indice_pairs
+ * and helper parameters: array \b indice_num, scaler \b inverse and \b sub_m.
+ *
+ * The tensors \b input_grad and \b output_grad is reordered from origin input
+ * gradient and output gradient. The tensor \b indice_pairs describes the
+ * calculation logic between the tensors \b input_grad and \b output_grad.
+ * Every pair in \b indice_pairs points to part of \b filters and the single
+ * line in \b input_grad and \b output_grad. Every single line in \b input_grad
+ * is calculated from matmul calculation logic with data mensioned above.
+ *
+ * @param[in] handle
+ * Handle to an MLUOP context that is used to manage MLU devices and queues in
+ * the IndiceConvolutionBackwardData operation. For detailed information, see
+ * ::mluOpHandle_t.
+ * @param[in] output_grad_desc
+ * The descriptor of input data \b output_grad, which contains dimension, data
+ * type and data layout.
+ * @param[in] output_grad
+ * Pointer to the MLU memory that stores the output_grad tensor. It is the
+ * gradient data of output tensor after reordered.
+ * @param[in] filters_desc
+ * The descriptor of input data \b filters, which contains dimension, data type
+ * and data layout. It contains N, H, W and C information when it is a 4D
+ * tensor, or N, D, H, W and C information when it is a 5D tensor.
+ * @param[in] filters
+ * Pointer to the MLU memory that stores the filter tensor.
+ * @param[in] indice_pairs_desc
+ * The descriptor of input data \b indice_pairs, which contains dimension,
+ * data type and data layout.
+ * @param[in] indice_pairs
+ * Pointer to the MLU memory that stores the indice_pairs tensor. It is used to
+ * specify the calculation pairs between \b input_grad and \b output_grad.
+ * @param[in] indice_num
+ * The array to describe the valid data number in \b indice_pairs.
+ * @param[in] inverse
+ * The param to describe the operation performs deconvolution logic or not.
+ * @param[in] sub_m
+ * The param to describe the operation performs sub_m convolution logic or
```

Comment:
The parameter to describe whether the operation performs sub_m convolution logic or